src/jdbc/cubrid/jdbc/jci/UStatement.java

```diff
@@ -129,6 +131,8 @@
 	private boolean result_cacheable = false;
 	private UStmtCache stmt_cache;
 
+        private ConnectionProperties connectionProperties = new ConnectionProperties();
```

Comment:
it is not used anywhere

Reply:
I checked the line is unnecessary. I modified it.